custom/auth/ante/fee.go

```diff
@@ -220,10 +257,22 @@ func (fd FeeDecorator) checkTxFee(ctx sdk.Context, tx sdk.Tx, taxes sdk.Coins) (
 		}
 
 		requiredFees := requiredGasFees.Add(taxes...)
+		allFees := requiredFees.Add(nonTaxableTaxes...)
 
 		// Check required fees
 		if !requiredFees.IsZero() && !feeCoins.IsAnyGTE(requiredFees) {
-			return 0, errorsmod.Wrapf(sdkerrors.ErrInsufficientFee, "insufficient fees; got: %q, required: %q = %q(gas) + %q(stability)", feeCoins, requiredFees, requiredGasFees, taxes)
+			// we don't have enough for tax and gas fees. But do we have enough for gas alone?
+			if !requiredGasFees.IsZero() && !feeCoins.IsAnyGTE(requiredGasFees) {
+				return 0, false, false, errorsmod.Wrapf(sdkerrors.ErrInsufficientFee, "insufficient fees; got: %q, required: %q = %q(gas) + %q(stability)", feeCoins, requiredFees, requiredGasFees, taxes)
```

Comment:
I think it should be `!feeCoins.IsAllGTE()` here, since we want to check if any provided `feeCoin` is insufficient for the gas fees